src/firefly/js/charts/ui/HistogramPlotly.jsx

```diff
@@ -156,7 +156,7 @@ export class HistogramPlotly extends Component {
             plotlyLayout: {
                 height,
                 width,
-                hovermode: 'x',
+                hovermode: 'compare',
```

Comment:
Please revert this change: there is no hovermode 'compare'.

Reply:
@tgoldina @lznakano was this revert before merged?